### PR TITLE
Use release-1.9 branch instead of master

### PR DESCRIPTION
**Pull request by jacob-delgado**
Change master to release-1.9
See https://github.com/istio/istio/issues/29490

### CODE REVIEW

**Review by brian-avery — APPROVED** [review]

File: release/build.sh [hunk 1]
```diff
@@ -62,17 +62,17 @@ ${DEPENDENCIES:-$(cat <<EOD
...
+    branch: release-1.9
     goversionenabled: true
   gogo-genproto:
     git: https://github.com/istio/gogo-genproto
-    branch: master
+    branch: release-1.9
   test-infra:
     git: https://github.com/istio/test-infra
-    branch: master
+    branch: release-1.9
```
Comment (howardjohn) [inline release/build.sh:72]: this should be master  still, we don't branch test-infra

File: test/publish.sh [hunk 2]
```diff
@@ -54,17 +54,17 @@ dependencies:
...
+    branch: release-1.9
     goversionenabled: true
   gogo-genproto:
     git: https://github.com/istio/gogo-genproto
-    branch: master
+    branch: release-1.9
   test-infra:
     git: https://github.com/istio/test-infra
-    branch: master
+    branch: release-1.9
```
Comment (howardjohn) [inline test/publish.sh:64]: same comment

**Review by howardjohn — CHANGES_REQUESTED** [review]
Please remove my hold whenever, just want to make sure it doesn't get lost in the noise of 100s of PRs

File: test/publish.sh (hunk 2 above)
Comment (jacob-delgado) [reply test/publish.sh]: Changed

File: release/build.sh (hunk 1 above)
Comment (jacob-delgado) [reply release/build.sh]: Changed

**Review by stevenctl — APPROVED** [review]

**Review by howardjohn — APPROVED** [review]

**Pull request merged**